docs/UserGuide.md

```diff
@@ -162,6 +162,44 @@ The list entry of the user with <id> will not have the tag anymore.
 
 ![delete-tag](images/user-guide/delete-tag_mock_output.png)
 
+### Filtering contacts : filter
+
+Filters the contacts.
+
+Format: `filter <component>[.<modifier>]/<value> ...`
+
+`component` is one of `n`,`p`,`e`, or `a` corresponding to the values in add:
```

Comment:
Maybe let the user have both the options of say using "phone" as well as "p" when they want to filter by phone number and so forth for the remaining keywords.

Reply:
Now that more friendly tag names are introduced, the syntax will likely change to something like `phone.is:12345678`. This will make the command more intuitive.

Of course, the command won't be as short as it was before, but I don't really think we need to shave keystrokes off of the program interface.